### PR TITLE
Rbh 66 user page

### PR DESCRIPTION
**Pull request by DavidPurdy1**
## Jira Ticket:
[RBH-66](https://roommatebudgethelper.atlassian.net/jira/software/projects/RBH/boards/1?selectedIssue=RBH-66)
## Description of Work
Added a user page with ability to add profile picture, signout, and set payment links for emails in the future. Notification settings for emails will also go here eventually. 
## Acceptance Criteria

- Add image and see if image appears above
- Edit Image and make verify image changes
- Verify that images look correct on billing and charging screens
- Add payment method, verify that payment method persists on refresh
- Change payment methods
- Sign out and verify that the user cannot access any pages

## How to Test

- Run e2e Cypress tests

### List of Changes:

- Changed how images look on screens showing image
- Return the signed url on queries, but store only the key
- Added 3 spots to store payment methods in the schema
- Page to set these options
- Sign out button 
- Future spot for the email notifications once that is set up

### Screenshots

| image | Top of page |
| image |
Bottom of Page | 

[RBH-66]: https://roommatebudgethelper.atlassian.net/browse/RBH-66?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by rhit-daviscl — CHANGES_REQUESTED** [review]
The JavaScript test failed. Needs the changes the test says to make

**Review by rhit-daviscl — CHANGES_REQUESTED** [review]
image
Whenever I try to update payment methods. I do not have any input fields.

File: app/src/pages/user.tsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    const setPaymentMethods = trpc.useMutation(["user.setPaymentMethods"]);
+    const {data: paymentMethods, refetch: refetchPaymentMethods} = trpc.useQuery(["user.getPaymentMethods"]);
+    const logOut = () => {
+        signOut();
+    };
+
+    const onUpdateNotificationSettings = async (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        const form = event.target as HTMLFormElement;
+        // TODO: Set notification settings for email here
```
[the remark this answers is not in the document]
Comment (RealJasomo) [reply app/src/pages/user.tsx:26]: Yeah let's do this as a separate task

**Review by RealJasomo — APPROVED** [review]
looks good, we can optimize the image loading here but we can do that as a separate issue

**Comment by DavidPurdy1** [comment]
> image Whenever I try to update payment methods. I do not have any input fields.

I haven't been able to recreate this issue. I am guessing that you might not have updated the schema/db. If you can try that when you have some time and let me know @rhit-daviscl

**Review by rhit-daviscl — APPROVED** [review]

**Pull request merged**